blosc/frame.c

```diff
@@ -914,17 +914,15 @@ uint8_t* get_coffsets(blosc2_frame *frame, int32_t header_len, int64_t cbytes, i
     return frame->coffsets;
   }
   if (frame->sdata != NULL) {
-    if (header_len + cbytes > frame->len) {
+    int32_t off_pos = header_len + cbytes;
+    // Check that there is enough room to read Blosc header
+    if (off_pos + BLOSC_MIN_HEADER_LENGTH > frame->len) {
```

Comment:
You should replace `BLOSC_MIN_HEADER_LENGTH` by `BLOSC_EXTENDED_HEADER_LENGTH`, as frames can only host extended (Blosc2) frames.

Reply:
Ok that should be fixed now.